### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by KANGPUNGYUN**
### Description

I translated the explanation of Aliquot Sum function into Korean.

### Checklist

- [x] A description of the changes in this PR is mentioned above.
- [  ] All the new and existing checks pass.
- [x] No plagiarized, duplicated, or repetitive documentation that has been directly copied from another source.
- [ ] If it's a new explanation, it contains solid, understandable, and accessible information.
- [x] I have read the whole [contributing guidelines](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/CONTRIBUTING.md) and agree to the [Code of Conduct](https://github.com/TheAlgorithms/.github/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by appgurueu — APPROVED** [review]
Looks fine (judging by Google Translate). We should probably just link to *The Algorithms* rather than individual sites.

File: ko/기초 수학/진약수의 합.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+이렇게 구한 합은 진약수의 합이다
+
+## 구현
+
+- [C#](https://github.com/TheAlgorithms/C-Sharp/blob/master/Algorithms/Numeric/AliquotSumCalculator.cs)
+- [Java](https://github.com/TheAlgorithms/Java/blob/master/src/main/java/com/thealgorithms/maths/AliquotSum.java)
+- [JavaScript](https://github.com/TheAlgorithms/JavaScript/blob/master/Maths/AliquotSum.js)
+- [Python](https://github.com/TheAlgorithms/Python/blob/master/maths/aliquot_sum.py)
+- [Ruby](https://github.com/TheAlgorithms/Ruby/blob/master/maths/aliquot_sum.rb)
```
Comment (Panquesito7) [inline ko/기초 수학/진약수의 합.md:46]: You might want to link the whole implementation website as well ([link here](https://the-algorithms.com/ko/algorithm/aliquot-sum)). 🙂

Comment (KANGPUNGYUN) [reply ko/기초 수학/진약수의 합.md:46]: That's right! Isn't it a link to _The Algorithms_? I thought this link was The _Algorithms link_. If it is not _The Algorithms_ site link, can I know the correct link address? Then i'll change the link :)

Comment (Panquesito7) [reply ko/기초 수학/진약수의 합.md:46]: It should be the link to [The Algorithms' website](https://the-algorithms.com) and [its implementation](https://the-algorithms.com/ko/algorithm/aliquot-sum).

Comment (appgurueu) [reply ko/기초 수학/진약수의 합.md:46]: Why to both? Just the link to the "The Algorithms page" of the algorithm (Aliquot Sum) should suffice.

Comment (Panquesito7) [reply ko/기초 수학/진약수의 합.md:46]: Edited.

**Review by KANGPUNGYUN — COMMENTED** [review]
I read [contributing guidelines](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/CONTRIBUTING.md) again. And I checked your latest commit(cycle sort.md) and changed it similarly. Is it OK?

**Review by Panquesito7 — APPROVED** [review]
Awesom. Thanks! 🚀

**Pull request merged**